### PR TITLE
[13563] Release lifecycle

### PR DESCRIPTION
**Pull request by JLBuenoLopez-eProsima**
Signed-off-by: JLBuenoLopez-eProsima <joseluisbueno@eprosima.com>

### CODE REVIEW

**Comment by JLBuenoLopez-eProsima** [comment]
@richiprosima please test this

**Comment by JLBuenoLopez-eProsima** [comment]
@richiprosima please test this

File: RELEASE_SUPPORT.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Release support
+
+Please, refer to the [master branch](https://github.com/eProsima/Fast-DDS/blob/master/RELEASE_SUPPORT.md) for the latest
+version of this document.
+
+*eProsima Fast DDS* maintains several releases with different support cycles.
+Major releases are scarce and they are made when public API is broken.
+Latest major release is v2.0.0 (June 2020) which introduced DDS compliant API.
```
Comment (EduPonz) [inline RELEASE_SUPPORT.md:8]: I'd add a link to the versioning policy document somewhere here

File: RELEASE_SUPPORT.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+Each quarter, a new Fast DDS minor version is released, including new features.
```
Comment (EduPonz) [inline RELEASE_SUPPORT.md:10]:
```suggestion
Each quarter, a new Fast DDS minor version with new features is released.
```

File: RELEASE_SUPPORT.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+By default, *eProsima Fast DDS* minor releases have a lifecycle of 6 months.
```
Comment (EduPonz) [inline RELEASE_SUPPORT.md:11]:
```suggestion
By default, *eProsima Fast DDS* minor releases have a lifecycle of **6 months**.
```

File: RELEASE_SUPPORT.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+However, the life of certain branches can be extended depending on costumer needs.
```
Comment (EduPonz) [inline RELEASE_SUPPORT.md:12]:
```suggestion
However, the life of certain branches can be extended depending on costumer and user needs.
```

File: RELEASE_SUPPORT.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+Within the support period of any minor version, there can be patch releases that either add new fuctionalities in an
```
Comment (EduPonz) [inline RELEASE_SUPPORT.md:13]:
```suggestion
Within the support period of any minor version, there may be several patch releases that either add new functionalities in an
```

File: RELEASE_SUPPORT.md [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+ABI compatible way, or fix possible issues.
```
Comment (EduPonz) [inline RELEASE_SUPPORT.md:14]: In general, only break lines after period in md

File: RELEASE_SUPPORT.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+Before the minor release end of life (EOL), a final patch release including the latest fixes will be done if there are
```
Comment (EduPonz) [inline RELEASE_SUPPORT.md:15]:
```suggestion
Before the minor release end of life (EOL) date, a final patch release including the latest fixes will be make available if there are any
```

File: RELEASE_SUPPORT.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+changes not included previously in any other patch release.
```
Comment (EduPonz) [inline RELEASE_SUPPORT.md:16]:
```suggestion
changes not included any previous patch release of the same minor version.
```

File: RELEASE_SUPPORT.md [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+|v1.8.x|May 2019|February 2022|
+|v1.9.x|August 2019|February 2022|
+|v1.10.x|April 2020|February 2022|
+|v2.0.x|June 2020|February 2022|
+|v2.1.x|November 2020|May 2023|
+|v2.2.x|January 2021|February 2022|
+|v2.3.x|March 2021|November 2022|
+|v2.4.x|September 2021|March 2022|
+|v2.5.x|December 2021|June 2022|
+|v2.6.x|March 2022|May 2023 [^*]|
```
Comment (EduPonz) [inline RELEASE_SUPPORT.md:32]: I'd order this from most recent to oldest

File: RELEASE_SUPPORT.md [hunk 10]
```diff
@@ -0,0 +1,34 @@
...
+
+[^*]: This release support could be extended.
```
Comment (EduPonz) [inline RELEASE_SUPPORT.md:34]:
```suggestion
[^*]: Support may be extended.
```

**Review by EduPonz — CHANGES_REQUESTED** [review]

File: RELEASE_SUPPORT.md [hunk 11]
```diff
@@ -0,0 +1,31 @@
+# Release support
+
+Please, refer to the [master branch](https://github.com/eProsima/Fast-DDS/blob/master/RELEASE_SUPPORT.md) for the latest version of this document.
+
+*eProsima Fast DDS* maintains several releases with different support cycles.
+Major releases are scarce and they are made when public API is broken (more information about *eProsima Fast DDS* versioning can be found [here](https://github.com/eProsima/Fast-DDS/blob/master/VERSIONING.md)).
+Latest major release is v2.0.0 (June 2020) which introduced DDS compliant API.
```
Comment (EduPonz) [inline RELEASE_SUPPORT.md:7]:
```suggestion
Latest major release is v2.0.0 (June 2020), which introduced the DDS compliant API.
```

File: RELEASE_SUPPORT.md [hunk 12]
```diff
@@ -0,0 +1,31 @@
...
+
+Each quarter, a new Fast DDS minor version with new features is released.
+By default, *eProsima Fast DDS* minor releases have a lifecycle of **6 months**.
+However, the life of certain branches can be extended depending on costumer and user needs.
+Within the support period of any minor version, there may be several patch releases that either add new functionalities in an ABI compatible way, or fix possible issues.
```
Comment (EduPonz) [inline RELEASE_SUPPORT.md:12]:
```suggestion
Within the support period of any minor version, there may be several patch releases that either add new functionalities in an ABI compatible way or fix possible issues.
```

**Review by EduPonz — CHANGES_REQUESTED** [review]

**Review by EduPonz — APPROVED** [review]
LGTM

File: RELEASE_SUPPORT.md [hunk 13]
```diff
@@ -0,0 +1,31 @@
...
+
+*eProsima Fast DDS* maintains several releases with different support cycles.
+Major releases are scarce and they are made when public API is broken (more information about *eProsima Fast DDS* versioning can be found [here](https://github.com/eProsima/Fast-DDS/blob/master/VERSIONING.md)).
+Latest major release is v2.0.0 (June 2020), which introduced the DDS compliant API.
+
+Each quarter, a new Fast DDS minor version with new features is released.
+By default, *eProsima Fast DDS* minor releases have a lifecycle of **6 months**.
+However, the life of certain branches can be extended depending on costumer and user needs.
+Within the support period of any minor version, there may be several patch releases that either add new functionalities in an ABI compatible way or fix possible issues.
+Before the minor release end of life (EOL) date, a final patch release including the latest fixes will be make available if there are any changes not included in any previous patch release of the same minor version.
```
Comment (MiguelCompany) [inline RELEASE_SUPPORT.md:13]:
```suggestion
Before the minor release end of life (EOL) date, a final patch release including the latest fixes will be made available if there are any changes not included in any previous patch release of the same minor version.
```

**Review by MiguelCompany — CHANGES_REQUESTED** [review]
Just a grammar issue.

**Review by richiware — DISMISSED** [review]

**Review by MiguelBarro — APPROVED** [review]
LGTM

**Review by EduPonz — APPROVED** [review]

**Review by MiguelCompany — APPROVED** [review]

**Pull request merged**